Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,6 +541,35 @@ bool isFunctionJson(string fullName)
             }
         }
 
+        private async Task LocalAppFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
+        {
```

Comment:
>{ [](start = 8, length = 1)

This method is very difficult to read. Group related lines together, separate unrelated lines with a blank line, add relevant comments (explaining what we are doing and why we are doing) wherever the code is not obvious.